app/furniture/furniture.rb

```diff
@@ -14,7 +14,15 @@ module Furniture
     spotlight: Spotlight,
   }.freeze
 
-  # Appends each {Furniture}'s CRUD actions
+  # Appends each {Furniture}'s CRUD actions under a FurniturePlacement
+  # @deprecated
+  def self.deprecated_append_routes(routing_context)
+    REGISTRY.each_value do |furniture|
+      furniture.deprecated_append_routes(routing_context) if furniture.respond_to?(:deprecated_append_routes)
+    end
+  end
+
+  # Appends each Furnitures CRUD actions within the {Room}
```

Comment:
Nitpick: On the line below, is the `routing_context` actually just the `router`? That seems to be at least the variable name you are using in e.g. `Marketplace.append_routes`. `router` seems clearer to me than `routing_context` -- I don't know what the latter is.